### DISCUSSION TITLE
Mounting n-filter fan

### DISCUSSION THREAD

**Discussion by Yanks1023** [gpstar81/GPStar-proton-pack#99] [General]
So I'm starting the n filter smoke mod done by Dustin. Can you elaborate on how you mounted the 50mm fan inside the n filter? 3d printed brackets? Glue? Thank you!

**Comment by gpstar81** [comment]
I will let Dustin comment on that, but an fyi, I merged our big update branch into the main branch, many new features, if you flash from this, make sure to update all your SD cards with the latest files.

Note: For Arduino Nano builds for the Neutrona Wand, please refer to the Flashing/Uploading code guide for changes you need to make to the serial transfer library before compiling and uploading. There is an additional step in the configuration code for the Arduino Mega builds as well, that is noted in the bottom of the code configuration file, you can't miss it. 

If you have problems, a **legacy-diy** branch has been created with the old release if you wish to still use that.

**Comment by DustinGrau** [comment]
Friction, mostly. A 50mm blower fan, mounted vertically, will just fit in the space under the N-filter and will be held in place by the bottom of the cyclotron interior assembly. Just drill a 1/2" hole where the blower's opening is and that will be sufficient. I did use some scrap EVA foam to keep the fan from twisting or turning, but again it was held in place by the existing parts. The benefit of this placement is that you only need to make another 1/4" hole for some airline tubing which will bring the smoke up into the cone of the N-filter.

**Comment by DustinGrau** [comment]
![IMG_2713](https://github.com/gpstar81/haslab-proton-pack/assets/115430732/360c3f57-73c7-49c0-97bd-accf2360ae73)

This shows the relative size of the blower which is close to the width of the cavity. You can see the tubing running through and a hole for the blower. Once holes are drilled, the order is to replace the cone on the N-filter and adjust the tubing, screw that back to the cyclotron housing, then insert the blower and position it before reassembling the cyclotron base.
